plane/source/docs/quadplane-frame-setup.rst

```diff
@@ -85,9 +88,21 @@ You can optionally move the quad motors to be on any other channel above
 
 Tricopter
 ---------
+Frame Type 7 is Tricopter and can be either non-Tiltrotor configuration, or :ref:`Tiltrotor<guide-tilt-rotor>` configured using either Vectored or Non-Vectored yaw control.
```

Comment:
its fine to add the default motor setups (its incorrect as shown), but changing the existing line 89 is not required...add it below

Reply:
I'm not sure how to say this Henry, but if by "the existing line 89" you mean this 

`Frame Type 7 is Tricopter and can be either non-Tiltrotor configuration, or :ref:`Tiltrotor<guide-tilt-rotor>` configured using either Vectored or Non-Vectored yaw control. If using non-Tiltrotor or Non-Vectored Yaw Tilt-rotor, the yaw control output is setup as Motor 7 (``SERVOn_FUNCTION`` = 39) using a tilt mechanism for the yaw motor, Motor 4. You should set up the yaw servo’s maximum lean angle in degrees with :ref:`Q_M_YAW_SV_ANGLE<Q_M_YAW_SV_ANGLE>`. This lean angle assumes that ``SERVOn_MIN`` and ``SERVOn_MAX``, represent +/- 90 degrees, with ``SERVOn_TRIM`` representing 0 degrees lean.`

It is unreadable. My changes are an attempt to make sense of it, because as it stands it is incredibly hard to decipher.